### PR TITLE
Fix formatting

### PR DESCRIPTION
**Pull request by ifdattic**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | 2.5 |
| Fixed tickets |  |

### CODE REVIEW

File: components/http_foundation/introduction.rst [hunk 1]
```diff
@@ -250,8 +250,8 @@ by using the following methods:
 :method:`Symfony\\Component\\HttpFoundation\\Request::getCharsets`
     Returns the list of accepted charsets ordered by descending quality.
 
-* :method:`Symfony\\Component\\HttpFoundation\\Request::getEncodings`:
-  returns the list of accepted charsets ordered by descending quality;
+:method:`Symfony\\Component\\HttpFoundation\\Request::getEncodings`
+    Returns the list of accepted charsets ordered by descending quality.
```
Comment (xabbuh) [inline components/http_foundation/introduction.rst]: We have to fix the sentence too (seems be copied from the one above). We are not talking about charsets here.

Comment (ifdattic) [inline components/http_foundation/introduction.rst]: Great catch!

**Comment by xabbuh** [comment]
:+1:

**Comment by WouterJ** [comment]
Thank you!

**Pull request closed without merging**